### PR TITLE
[NOT AZURE] Azure Stack Azure Bridge Admin SDK

### PR DESCRIPTION
**Pull request by deathly809**
## Description

This is for Azure Stack **ONLY**.  There seems to be an issue when generating the SDK remotely for the repo some reason.  I have message the Autorest team and ask them for help but no response.

https://github.com/Azure/azure-rest-api-specs/pull/2781

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [X] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/_metadata/azsadmin_resource-manager_compute.txt [hunk 1]
```diff
@@ -1,11 +1,2 @@
-2018-04-11 18:34:50 UTC
+﻿
```
Comment (dsgouda) [inline src/SDKs/_metadata/azsadmin_resource-manager_compute.txt]: Please generate the code using generate.ps1 or generate.cmd and commit those changes.
Looks like autorest failed to generate the code from config file here

File: src/SDKs/_metadata/azsadmin_resource-manager_azurebridge.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/azsadmin/resource-manager/azurebridge/readme.md --csharp --csharp-sdks-folder=E:\github\azure-sdk-for-net\src\AzureStack\AzureBridgeAdmin\AzureBridge.Admin --version=latest --reflect-api-versions
+
```
Comment (dsgouda) [inline src/SDKs/_metadata/azsadmin_resource-manager_azurebridge.txt]: Please generate the code using generate.ps1 or generate.cmd and commit those changes.
Looks like autorest failed to generate the code from config file here

File: src/SDKs/_metadata/azsadmin_resource-manager_compute.txt (hunk 1 above)
Comment (dsgouda) [reply src/SDKs/_metadata/azsadmin_resource-manager_compute.txt]: Please checkout this file and run the code generation script again. These changes are not expected.

File: src/AzureStack/AzureBridgeAdmin/AzureBridge.Admin/Microsoft.AzureStack.Management.AzureBridge.Admin.csproj [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    <PackageReleaseNotes>This is a preview release of the AzureStack AzureBridge Admin SDK.</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="node_modules\JSONSelect\site\css\droid_sans.tt">
+      <Generator>TextTemplatingFileGenerator</Generator>
+    </None>
+  </ItemGroup>
```
Comment (dsgouda) [inline src/AzureStack/AzureBridgeAdmin/AzureBridge.Admin/Microsoft.AzureStack.Management.AzureBridge.Admin.csproj]: Add [these](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj#L23) lines at the bottom

File: src/AzureStack/AzureBridgeAdmin/AzureBridge.Admin.Tests/src/DownloadedProductTests.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            RunTest((client) => {
+                var productName1 = "Canonical.UbuntuServer1710-ARM.1.0.6";
+                var product = client.DownloadedProducts.Get("azurestack-activation", "default", productName1);
+                Assert.NotNull(product);
+            });
+        }
+        /*
+        [Fact]
+        public void TestRemoveAzsAzureBridgeDownloadedProduct()
+        {
```
Comment (dsgouda) [inline src/AzureStack/AzureBridgeAdmin/AzureBridge.Admin.Tests/src/DownloadedProductTests.cs]: nit: was this intended

File: src/AzureStack/AzureBridgeAdmin/AzureBridge.Admin/Microsoft.AzureStack.Management.AzureBridge.Admin.csproj [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    <VersionPrefix>0.1.0-preview</VersionPrefix>
+    <AssemblyName>Microsoft.AzureStack.Management.AzureBridge.Admin</AssemblyName>
+    <PackageTags>Microsoft Azure Stack;AzureBridge;REST HTTP client;azureostackfficial;netcore451511</PackageTags>
+    <PackageReleaseNotes>This is a preview release of the AzureStack AzureBridge Admin SDK.</PackageReleaseNotes>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <None Update="node_modules\JSONSelect\site\css\droid_sans.tt">
```
Comment (dsgouda) [inline src/AzureStack/AzureBridgeAdmin/AzureBridge.Admin/Microsoft.AzureStack.Management.AzureBridge.Admin.csproj]: Why is this needed

**Review by dsgouda — CHANGES_REQUESTED** [review]
Left a few comments that need addressing.

**Comment by deathly809** [comment]
@dsgouda I have regenerated/rebuilt but the metadata file is empty.

**Review by dsgouda — APPROVED** [review]
LGTM subject to builds passing

**Comment by deathly809** [comment]
@dsgouda Passed!
